src/logger/advanced-outputs.ts

```
@@ -0,0 +1,333 @@
+      this.sendHttpRequestNodeJS(payload);
+    }
+  }
+
+  /**
+     * Fallback HTTP implementation for Node.js environments without fetch
+     */
+  private sendHttpRequestNodeJS(payload: any): void {
+    try {
+      const https = require('https');
+      const http = require('http');
+      const url = require('url');
```

ESLint

[error] 'url' is assigned a value but never used.

(@typescript-eslint/no-unused-vars)